base/tps/src/org/dogtagpki/server/tps/processor/TPSPinResetProcessor.java

```diff
@@ -49,7 +49,17 @@ public TPSPinResetProcessor(TPSSession session) {
 
     @Override
     public void process(BeginOpMsg beginMsg) throws TPSException, IOException {
-        if (beginMsg == null) {
+
+        IConfigStore configStore = CMS.getConfigStore();
+        String configName = "op.pinReset.testNoBeginMsg";
+        boolean testPinResetNoBeginMsg = false;
+        try {
+            testPinResetNoBeginMsg = configStore.getBoolean(configName);
```

Comment:
I think if you assign a default to false at time of getBoolean it'd be less expensive as it wouldn't have to throw an extra extension at regular run time.